tests/python/unittest/test_sparse_operator.py

```diff
@@ -1638,10 +1638,10 @@ def check_sparse_elementwise_sum_with_shape(stype, shape, n):
 @with_seed()
 def test_sparse_embedding():
     ''' test sparse embedding operator '''
-    def check_sparse_embedding(in_dim, out_dim, batch, densities, deterministic):
+    def check_sparse_embedding(in_dim, out_dim, batch, densities, deterministic, weight_stype):
```

Comment:
What about change the name to "check_deterministic"? "deterministic" does not sound really straightforward.

Reply:
deterministic is the param passed to mx.sym.contrib.SparseEmbedding. See line 1645